src/buttons/Chip.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { TouchableWithoutFeedback, StyleSheet } from 'react-native';
+import { withTheme } from '../config';
+import Button, { ButtonProps } from './Button';
+
+export type ChipProps = Omit<
+  ButtonProps,
+  'loading' | 'loadingStyle' | 'loadingProps'
+> & {
+  type?: 'solid' | 'outline';
+};
+
+const Chip: React.FunctionComponent<ChipProps> = (props: ChipProps) => {
+  return (
+    <Button
+      titleStyle={StyleSheet.flatten([
+        { fontSize: 14, paddingHorizontal: 2 },
+        props.titleStyle,
+      ])}
+      buttonStyle={StyleSheet.flatten([
+        { borderRadius: 30 },
+        props.buttonStyle,
```

Comment:
You have to separate out `buttonStyle` and `titleStyle` from `props` or else, the style gets overwritten below when `{...props}` is used. Separate it out in the parameters list like other components.

Reply:
All done! Kindly review